app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,11 @@
   </head>
 
   <body>
+    <% flash.each do |type, msg| %>
```

Comment:
Where is `type` referenced in this flash .each block? Is it necessary?

Reply:
nope, "type" was unnecessary and i accidentally copied it over from my other project.